app/controllers/projects_controller.rb

```diff
@@ -40,19 +42,22 @@ def edit
   def update
     if @project.update(project_params)
       flash[:success] = t('flash_messages.update', name: t('shared.project'))
+      respond_to do |format|
+        format.js { redirect_to projects_path }
+      end
     else
-      flash[:danger] = t('flash_messages.error', error_msg: @project.errors.full_messages.first)
-    end
-    respond_to do |format|
-      format.js { redirect_to projects_path }
+      respond_to do |format|
+        format.js
+      end
     end
   end
 
   def show
```

Comment:
To display the list of teams, use the same partial of teams table this is used on teams index page. Similarly for issues and users.

Reply:
will do in next PR